web/app/js/components/Namespaces.jsx

```diff
@@ -0,0 +1,127 @@
+import _ from 'lodash';
+import CallToAction from './CallToAction.jsx';
+import ConduitSpinner from "./ConduitSpinner.jsx";
+import ErrorBanner from './ErrorBanner.jsx';
+import MetricsTable from './MetricsTable.jsx';
+import PageHeader from './PageHeader.jsx';
+import { processRollupMetrics } from './util/MetricUtils.js';
+import React from 'react';
+import './../../css/list.css';
+import 'whatwg-fetch';
+
+export default class Namespaces extends React.Component {
```

Comment:
something to consider down the road, the namespace-specific pages have no reference/context in the navigation. consider a tree or some other way to indicate to the user where they are, and how they could navigate to/from that page.

Reply:
Yeah! I've added a namespace index page per your other comment; so now you can navigate to individual namespace pages via the ServiceMesh page or via the Namespace index page at `/namespaces`